app/assets/javascripts/host_edit.js

```diff
@@ -1,5 +1,12 @@
 //= require parameter_override
 
+var HOST_EDIT_ATTRIBUTES = {
+  "architecture" : ['architecture_id', 'organization_id', 'location_id'],
```

Comment:
nitpick: why dont write it without the strings, e.g.

```js
var HOST.. = {
  architecture: [...],
}
```
then simply access it by
```js
HOST_EDIT_ATTRIBUTES.architecture
```